jsk_fetch_robot/fetcheus/fetch-interface.l

```diff
@@ -40,8 +40,11 @@
      (ros::warn ":angle-vector tm is not a number, use :angle-vector av tm args"))
    (let ((use-torso t))
      (if (and (member :use-torso args) (null (cadr (member :use-torso args)))) (setq use-torso nil))
-     (send self :angle-vector-motion-plan av :move-arm :rarm :total-time tm :start-offset-time 0.01 :use-torso use-torso :clear-velocities t)
+     (send* self :angle-vector-motion-plan av :move-arm :rarm :total-time tm :start-offset-time 0.01 :use-torso use-torso :clear-velocities t args)
    ))
+  (:rarm-angle-vector 
+   (av &optional (tm 3000) &rest args)
+   (send* self :angle-vector av tm :ctype :arm-controller args))
```

Comment:
it is not needed, but I thought it may be useful for when one wants to move the arm without changing the camera's point of view. 
Doesn't seem to have a great demand so maybe doesn't really pay off defining a new method for it, but also didn't see any down-points for letting it defined.

Just realized now, but need to adjust the torso-controller according to the :use-torso value. (probably updating this later)